rhino/src/main/java/org/mozilla/javascript/Slot.java

```diff
@@ -82,7 +82,7 @@ int getAttributes() {
         return attributes;
     }
 
-    synchronized void setAttributes(int value) {
+    void setAttributes(int value) {
```

Comment:
I don't understand why you should use "synchronized" here. `attributes` is not a long or double and write should be atomic: https://docs.oracle.com/javase/specs/jls/se9/html/jls-17.html#jls-17.7

(even if it is a long/double, it would be atomic on 64bit jvms)